Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -87,6 +87,10 @@ def genGridOnPoints(data1,gm,deep=True,grid=None,geo=None):
     if grid is None:
       x = g.getLongitude()[:]
       y = g.getLatitude()[:]
+      xm = x[0]
+      xM = x[-1]
+      ym = y[0]
```

Comment:
@doutriaux1 I would suggest you use 

xMin or x_min and xMax or x_max. Var names does not have to be too long but atleast somewhat more intuitive as not only it will improve the readability it will help debugging bugs faster.